package.json

```diff
@@ -13,6 +13,7 @@
     "start:mmi": "yarn start --build-type mmi",
     "start:lavamoat": "yarn build:dev dev --apply-lavamoat=true",
     "dist": "yarn build dist",
+    "dist:mv3": "ENABLE_MV3=true yarn build dist",
```

Comment:
Maybe we should move this to a separate MR to make sure the functionality is related?